src/lib/dnssd/Discovery_ImplPlatform.h

```diff
@@ -83,8 +84,8 @@ class DiscoveryImplPlatform : public ServiceAdvertiser, public Resolver
     bool mIsCommissionerPublishing       = false;
     uint8_t mCommissionableInstanceName[sizeof(uint64_t)];
 
-    bool mDnssdInitialized               = false;
-    ResolverDelegate * mResolverDelegate = nullptr;
+    bool mDnssdInitialized = false;
+    ResolverDelegate * mResolverDelegates[CHIP_CONFIG_MDNS_MAX_RESOLVER_DELEGATES];
```

Comment:
```suggestion
    ResolverDelegate * mResolverDelegates[CHIP_CONFIG_MDNS_MAX_RESOLVER_DELEGATES] = {};
```
I know the object is static, but better to initialize it explicitly.

Reply:
Done.